quint/src/runtime/impl/compilerImpl.ts

```diff
@@ -984,7 +1031,7 @@ export class CompilerVisitor implements IRVisitor {
     // this function gives us access to the compiled operator later
     let callableRef: () => Maybe<Callable>
 
-    if (lookupEntry === undefined || lookupEntry.kind !== 'param') {
+    if (lookupEntry.kind !== 'param') {
```

Comment:
I removed the `lookupEntry === undefined` case because it is impossible (see the last if a few lines above)